lib/virt_autotest/utils.pm

```diff
@@ -159,6 +159,7 @@ sub is_kvm_host {
 
 #retrun 1 if libvirt 9.0- is running which monolithic libvirtd is the default service
 sub is_monolithic_libvirtd {
+    record_info('WARNING', 'Libvirt package is not installed', result => 'fail') if (script_run('rpm -q libvirt-libs'));
```

Comment:
I still think there is no need to give a "fail" result even for record_info inside a ```is_xxx``` function. For example, ```is_xxx``` function only responsible for answering "yes" or "no". If a system just need to know whether it is monolithic_libvirtd, but it gets a "fail" flag, this is not very reasonable. Any decision, including recording a failure info, should be done outside of ```is_xxx```.

Reply:
this function is not a tempoary function to differentiate it is modular libvirt. it will be obsolete when modular libvirt becomes to be the default virtualization. I even had thought add a setting 'IS_MONOLITHIC_LIBVIRT=1' for hosts with older products.

We know It is not perfect function, because there are 3 possible results, 'yes', 'no' and 'no libvirt'. but it has been widely used in virt tests as we overlooked the 'no libvirt'. I think it is the best solution with minimal effort currently we can find out as SLEM 5.5 is waiting for green tests.

We can think of making it better in the future when it is really needed.

Reply:
OK. Fine.